lib/rspec/rails/view_path_builder.rb

```diff
@@ -0,0 +1,53 @@
+module RSpec
+  module Rails
+    # Builds paths for view specs using a particular route set.
+    class ViewPathBuilder
+      def initialize(route_set)
+        @routes = route_set.routes
+        self.class.send(:include, route_set.url_helpers)
```

Comment:
After a 2nd look at the class, is this used somewhere in the class? Is this how we're getting access to `url_for`?